lib/utils/scroll_behavior.dart

```diff
@@ -61,7 +61,7 @@ class ClampingScrollWrapper extends StatelessWidget {
   Widget build(BuildContext context) {
     return ScrollConfiguration(
       behavior: ClampingScrollBehavior(),
-      child: child,
+      child: child!,
```

Comment:
As `ScrollConfiguration` requires a child, maybe `ClampingScrollWrapper` should as well? Or at least check for nulabillity before using the bang operator, returning a `SizedBox()` otherwise.

Reply:
Good call!